cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/PartitionSizeAnomalyFinder.java

```diff
@@ -108,9 +108,12 @@ public void configure(Map<String, ?> configs) {
     String topicExcludedFromCheck = (String) configs.get(TOPIC_EXCLUDED_FROM_PARTITION_SIZE_CHECK);
     _topicExcludedFromCheck = Pattern.compile(topicExcludedFromCheck == null ? DEFAULT_TOPIC_EXCLUDED_FROM_PARTITION_SIZE_CHECK
                                                                              : topicExcludedFromCheck);
-    Integer partitionSizeThreshold = (Integer) configs.get(SELF_HEALING_PARTITION_SIZE_THRESHOLD_MB_CONFIG);
-    _partitionSizeThresholdInMb = partitionSizeThreshold == null ? DEFAULT_SELF_HEALING_PARTITION_SIZE_THRESHOLD_MB
-                                                                 : partitionSizeThreshold;
+    String partitionSizeThresholdStr = (String) configs.get(SELF_HEALING_PARTITION_SIZE_THRESHOLD_MB_CONFIG);
+    _partitionSizeThresholdInMb = DEFAULT_SELF_HEALING_PARTITION_SIZE_THRESHOLD_MB;
+    if (partitionSizeThresholdStr != null) {
+      _partitionSizeThresholdInMb = Integer.parseInt(partitionSizeThresholdStr.trim());
+    }
```

Comment:
```suggestion
    _partitionSizeThresholdInMb = partitionSizeThreshold == null ? DEFAULT_SELF_HEALING_PARTITION_SIZE_THRESHOLD_MB
                                                                 : Integer.parseInt(partitionSizeThresholdStr.trim());
```
Preserves style, similar to [TopicReplicationFactorAnomalyFinder](https://github.com/linkedin/cruise-control/blob/main/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/TopicReplicationFactorAnomalyFinder.java#L251).
Personally I'd also remove the `trim()` to maintain the same behaviour as other numerical configs.

Reply:
@k0b3rIT please apply the suggestion

Reply:
Sure, I have applied it!